src/components/calcite-value-list-item/calcite-value-list-item.tsx

```diff
@@ -84,22 +95,51 @@ export class CalciteValueListItem {
 
   getPickListRef = (el) => (this.pickListItem = el as HTMLCalcitePickListItemElement);
 
+  handleKeyDown = (event: KeyboardEvent) => {
+    if (event.key === " ") {
+      this.handleActivated = !this.handleActivated;
+    }
+  };
+
+  handleBlur = () => {
+    this.handleActivated = false;
+  };
+
   // --------------------------------------------------------------------------
   //
   //  Render Methods
   //
   // --------------------------------------------------------------------------
 
+  renderHandle() {
+    const { icon } = this;
+    if (icon === ICON_TYPES.grip) {
+      return (
+        <span
+          role="button"
+          class={{ [CSS.handle]: true, [CSS.handleActivated]: this.handleActivated }}
+          tabindex="0"
+          data-js-handle="true"
+          aria-pressed={this.handleActivated}
+          onKeyDown={this.handleKeyDown}
+          onBlur={this.handleBlur}
+        >
+          <CalciteIcon size="16" path={drag16} />
+        </span>
+      );
+    }
+  }
+
   render() {
     return (
-      <Host>
+      <Host data-id={this.guid}>
```

Comment:
fixme: I'm hesitant about polluting the dom with data attributes unless the data-attribute is within the shadow dom. If we need it to be a property can we just setup an `id` or `guid` prop for the unique identification?

Reply:
This is what sortable uses by default to identify items for sorting.
https://github.com/SortableJS/Sortable#options
See `dataIdAttr`

Reply:
Ok. I guess this can be an exception.